microovn/node/node.go

```diff
@@ -286,6 +287,9 @@ func leaveCentral(ctx context.Context, s state.State) error {
 		logger.Warnf("Failed to get SB database specification: %s", err)
 	}
 
+	os.Remove(paths.CentralDBNBPath())
```

Comment:
Looking at the commit message of this commit, while I see these changes mentioned briefly at the end, I would expect only test changes and no code changes.

This is a prominent change in behavior which together with the `path.go` changes belongs in a separate commit with solid reasoning as to why this is necessary and why the chosen approach is safe.